ampersand-view.js

```diff
@@ -297,6 +289,22 @@ assign(View.prototype, {
         this.on('change', action, this);
     },
 
+    // Parser the declarative subview definition.
+    // Overload this method to create your own declarative subview style
+    _parseSubviewOpts: function (subview, name) {
```

Comment:
It has no purpose.  I originally removed it but then it got added back in the shuffle.